controllers/eventhandlers/service.go

```diff
@@ -14,43 +13,44 @@ import (
 
 	"k8s.io/client-go/util/workqueue"
 	"sigs.k8s.io/controller-runtime/pkg/event"
-	gateway_api "sigs.k8s.io/gateway-api/apis/v1beta1"
 	mcs_api "sigs.k8s.io/mcs-api/pkg/apis/v1alpha1"
 )
 
-type enqueueRequetsForServiceEvent struct {
+type enqueueRequestForServiceWithExportEvent struct {
+	log    gwlog.Logger
 	client client.Client
 }
 
-func NewEqueueRequestServiceEvent(client client.Client) handler.EventHandler {
-	return &enqueueRequetsForServiceEvent{
+func NewEqueueRequestServiceWithExportEvent(log gwlog.Logger, client client.Client) handler.EventHandler {
+	return &enqueueRequestForServiceWithExportEvent{
```

Comment:
I think  both old name and new name  are confusing... Can we change to a name likes  `NewK8sServiceEventHandlerForServiceExportController`? (or do it in a separate PR)